### PR TITLE
add a simple observer readme section

### PR DESCRIPTION
**Pull request by 43081j**
Updated the readme a tiny bit to explain how to do a simple observer (a property's observer).

would typescript complain about this? doesn't seem to locally but wasn't sure about accessing a private method on the prototype.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,14 @@ class TestElement extends Polymer.Element {
   @property()
   bar: string = 'yes';
 
+  // @observe does not support simple observers,
+  // you can already do that via a normal property
+  @property({ observer: TestElement.prototype.onBazChanged })
```
Comment (aomarks) [inline README.md]: This depends on https://github.com/Polymer/polymer-decorators/pull/18 right? That requires that `observer` is a `string`, which is also how Polymer normally works (AFAIK you can't pass a function reference in the normal Polymer block, or at least that's an undocumented feature if you can).

Comment (43081j) [reply README.md]: It depends on #18 for the type definitions (`PropertyOptions` has no `observer` in master).

You can pass a function reference as of very recently due to polymer/polymer#4574. it doesn't seem to be documented well, if at all.

its probably best we recommend this, though, as its a little stronger typed than the string version.

Comment (aomarks) [reply README.md]: > It depends on #18 (PropertyOptions has no observer in master). so yes we need to change it to string|((val: any, old: any) => void) or something i suppose, maybe in this PR?
> 
> You can pass a function reference as of very recently due to polymer/polymer#4574. it doesn't seem to be documented well, if at all.

Ah, I missed that change. Yes, would be great to also add the new signature to this PR, `string|((val: any, old: any) => void)` seems right.

BTW we should also update https://github.com/Polymer/polymer/blob/master/types/extra-types.d.ts to match https://github.com/Polymer/polymer/blob/master/externs/polymer-externs.js#L24, maybe in https://github.com/Polymer/polymer/pull/5000?

Comment (43081j) [reply README.md]: yup can do. i added the `observer` definition here too now.

**Pull request merged**